examples/react-17/src/Highlight.tsx

```diff
@@ -11,7 +11,7 @@ type HighlightHitParams<THit> = {
    *
    * You can use the array syntax to reference nested attributes.
    */
-  attribute: keyof THit | string[];
+  attribute: keyof THit | (string | number)[];
```

Comment:
An alternative would be to add `@algolia/autocomplete-shared` as a dependency in this example project, and import the existing type, so this doesn't need to be duplicated